po/glossary/gnc-glossary.txt

```diff
@@ -1,212 +1,212 @@
-"vendor"	"In small business accounting: A person or company that sells items and is supplying goods"
-"voucher"	"The terms 'Voucher' and 'Expense Voucher' are used interchangeably in gnucash. The 'Expense Voucher' is also a bit of a misnomer -- it's more like an 'Expense Report' in gnucash.  The phrase is meant to be a list of expenses incurred by an employee for which the company will reminburse them."
-"withdraw (in the reconcile dialog)"	"see debit"
+"Context [optional]"	"Term"	"English Definition or explanation. Purpose of this record: document the file format for maintainers—no need to translate it."
+"Quoting convention"	"\"\""	"Opening and closing quote symbols and optionally their key combos like [altgr]+[Y]/[X]. Define the preferred style of quotation, see https://en.wikipedia.org/wiki/Quotation_mark#Summary_table"
+	"account"	"A detailed record of money spent and received"
+	"account code"	"An alphanumerical key of an account in GnuCash, not at the bank, can be used to sort. Some templates provide them or the user can enter them."
+	"account hierarchy"	"the tree view of all accounts"
+	"account name"	"-"
+"account type"	"Active"	"The left side of the balance sheet in T account form shows the application of funds in form of assets. Because it contains only assets use assets directly. Complement: Passive. See also: Report Form"
+"account type"	"Asset"	"A thing, esp. owned by a person or company, that has value and can be used or sold to pay debts. Dependent on the context you might use 'account type: Active' instead."
+"account type"	"Assets & Liabilities"	"in fact: 'Active & Passive', group aka 'Balance Sheet accounts'; complement of 'Profit & Loss'"
+"account type"	"checking"	"(esp. US) (Brit = current account) a bank account from which money can be withdrawn without previous notice"
+"account type"	"currency"	"-"
+"account type"	"Equity"	"see: Equity, but see also 'account type: Passive'"
+"account type"	"Expense"	"-"
+"account type"	"Income"	"-"
+"account type"	"Liability"	"A debt, a financial obligation, but see also 'account type: Passive'"
+"account type"	"money-market"	"-"
+"account type"	"Mutual fund"	"-"
+"account type"	"Passive"	"The right side of the balance sheet in T account form shows the source of funds and contains equity & liability. While not common in english, most languages would translate 'equity & liability' with 'passive'. Complement: Active. See also: Report Form Implementation: https://bugs.gnucash.org/show_bug.cgi?id=421766"
+"account type"	"Profit & Loss"	"Group of accounts tracking your success, complement of 'Assets & Liabilities'"
+"account type"	"saving"	"1. (US) any type of account that earns interest 2. (Brit) any type of bank account that earns a higher level of interest than a current account or deposit account"
+"account type"	"Stock"	"-"
+"account type"	"trading"	"This account type (new in gnucash-2.4.0) is used when exchanging or trading amounts from one currency into another"
```

Comment:
We can lose the aside about GnuCash 2.4, that was long enough ago that no translator will care.